mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowProtobufMapper.java

```diff
@@ -160,8 +160,16 @@ String makeCreateModelVersion(String modelName, String runId, String source) {
     return print(builder);
   }
 
-  String makeGetModelVersionDetails(String modelName, String version) {
-    return print(GetModelVersion.newBuilder().setName(modelName).setVersion(version));
```

Comment:
good thing tests were added, turns out this is syntax is used for post requests and get requests